libs/cms/src/lib/cms.resolver.ts

```diff
@@ -444,6 +449,33 @@ export class CmsResolver {
     )
   }
 
+  @CacheControl(defaultCache)
+  @Query(() => GrantList)
+  async getGrants(@Args('input') input: GetGrantsInput): Promise<GrantList> {
+    return this.cmsElasticsearchService.getGrants(
+      getElasticsearchIndex(input.lang),
+      input,
+    )
+  }
+
+  @CacheControl(defaultCache)
+  @Query(() => Grant, { nullable: true })
+  async getSingleGrant(
+    @Args('input') { lang, id }: GetSingleGrantInput,
+  ): Promise<(Partial<Grant> & { lang: Locale }) | null> {
+    const grant: Grant | null = await this.cmsContentfulService.getGrant(
+      lang,
+      id,
+    )
+
+    if (!grant) return null
+
+    return {
+      ...grant,
+      lang,
```

Comment:
Is there a reason why you are also passing down the locale? I know this is something done for articles but that's because the locale is used in the field resolver

Reply:
No, removed